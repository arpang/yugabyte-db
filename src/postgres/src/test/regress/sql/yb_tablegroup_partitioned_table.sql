CREATE DATABASE test_tablegroup_partitioned_tables;
\c test_tablegroup_partitioned_tables
CREATE TABLEGROUP tg1;
CREATE TABLE prt (id int PRIMARY KEY, v int) PARTITION BY RANGE (id) TABLEGROUP tg1;
CREATE TABLE prt_p1 PARTITION OF prt FOR VALUES FROM (1) TO (4);
\dgrt
CREATE TABLE prt_p2 PARTITION OF prt FOR VALUES FROM (5) TO (9) TABLEGROUP tg1;