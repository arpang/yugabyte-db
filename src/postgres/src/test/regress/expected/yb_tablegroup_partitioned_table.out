CREATE DATABASE test_tablegroup_partitioned_tables;
\c test_tablegroup_partitioned_tables
CREATE TABLEGROUP tg1;
CREATE TABLE prt (id int PRIMARY KEY, v int) PARTITION BY RANGE (id) TABLEGROUP tg1;
CREATE TABLE prt_p1 PARTITION OF prt FOR VALUES FROM (1) TO (4);
\dgrt
              List of tablegroup tables
 Group Name | Group Owner |  Name  | Type  |  Owner   
------------+-------------+--------+-------+----------
 tg1        | yugabyte    | prt    | table | yugabyte
 tg1        | yugabyte    | prt_p1 | table | yugabyte
(2 rows)

CREATE TABLE prt_p2 PARTITION OF prt FOR VALUES FROM (5) TO (9) TABLEGROUP tg1;
ERROR:  syntax error at or near "TABLEGROUP"
LINE 1: ...rt_p2 PARTITION OF prt FOR VALUES FROM (5) TO (9) TABLEGROUP...
                                                             ^